lua/telescope/builtin.lua

```diff
@@ -904,4 +904,55 @@ builtin.marks = function(opts)
   }):find()
 end
 
+builtin.tags = function(opts)
+  -- TODO(conni2461): Handle opts like no filename, etc
+  opts = opts or {}
+
+  -- TODO(conni2461): Do we have a function like this already?! In plenary for example
+  -- If not put in utils or plenary
+  local function file_exists(file)
```

Comment:
There is builtin function 

```
if vim.fn.filereadable(vim.fn.exapnd(filepath)) == 1 then
   print("file exists, do stuff")
end

```

```

Reply:
@Conni2461

Reply:
Thanks that saved me a lot of time :smile: